game_event/views.py

```diff
@@ -150,3 +166,34 @@ def process_answer_game_event(request, id):
     content['event_exists'] = True
 
     return render(request, 'game_event/result.html', content)
+
+
+def get_weather_from_api(game_events):
+    weather = {}
+    for game in game_events:
+        url = 'https://api.openweathermap.org/data/2.5/weather'
+        params = {
+            'q': game.court.city,
+            'units': 'metric',
+            'appid': 'bd708048ad9d3ee449660bedad49f30e'
```

Comment:
be aware that in general exposing the api key I the code is really bad from security point of view.

Reply:
Hey, @SchSeba I'm aware of it, I also got a warning from 'GitGuardian', But I don't know what is a good practice in that case. I would use a different static file with all of the app secrets, and also this api key, and add this file to .gitignore file, so it would be available locally to me and wouldn't be exposed. But in this case how would the requests will work for the rest of our team? For example in this case, where I could the appid stored it unshared different file: 
`params = {
            'q': game.court.city,
            'units': 'metric',
            'appid': weatherSecret`

The flake8 and pytest would probably won't work, and if will work, than the http request will fail.

Reply:
Would be great if someone can elaborate on that issue

Reply:
@yftacherzog @Yuval-Vino @SchSeba

Reply:
I think you can load all the secret you need in the settings file from django by using environment variables.
then you can share with the team what are the env variables needed to run the application

Reply:
After a call with @yftacherzog, @IdanZimi and I removed the explicitly stated API Key, and change it to 'appid': os.getenv("APPID"), so that when running the server, we should attach the API-KEY ad environment variable, like this :
APPID='api-key' pipenv run python manage.py runserver 0.0.0.0:8000.
we also changed the view function, so if someone run the server without attaching the api key, pipenv run python manage.py runserver 0.0.0.0:8000 there won't be any problems, just that the game events table, will render without it.